examples/mplot3d/box3d.py

```diff
@@ -0,0 +1,75 @@
+"""
+===================
+3D box surface plot
+===================
+
+Given data on a gridded volume ``X``, ``Y``, ``Z``, this example plots the
+data values on the volume surfaces.
+
+The strategy is to select the data from each surface and plot
+contours separately using `.Axes3D.contourf` with appropriate
```

Comment:
There are two of these apparently:
```
/home/circleci/project/doc/gallery/mplot3d/box3d.rst:28: WARNING: more than one target found for cross-reference 'Axes3D.contourf': mpl_toolkits.mplot3d.axes3d.Axes3D.contourf, mpl_toolkits.mplot3d.Axes3D.contourf
```
```suggestion
contours separately using `.axes3d.Axes3D.contourf` with appropriate
```